server/app/schema.py

```diff
@@ -954,72 +953,65 @@ def resolve_items(self, info, **kwargs):
     )
 
     def resolve_sets(self, info, **kwargs):
-        with session_scope() as db_session:
-            locale = str(get_locale())
-            filters = kwargs.get("filters")
-            set_query = (
-                db_session.query(ModelSet)
-                .join(ModelSetTranslation)
-                .filter_by(locale=locale)
-            )
+        locale = str(get_locale())
+        filters = kwargs.get("filters")
+        set_query = (
+            db.session.query(ModelSet)
+            .join(ModelSetTranslation)
+            .filter_by(locale=locale)
+        )
 
-            level_sq = (
-                db_session.query(
-                    ModelItem.set_id, func.max(ModelItem.level).label("level")
-                )
-                .group_by(ModelItem.set_id)
-                .subquery()
-            )
-            set_query = set_query.join(level_sq, ModelSet.uuid == level_sq.c.set_id)
-
-            if filters:
-                search = filters.search.strip()
-                if filters.stats:
-                    set_query = set_query.join(ModelSetBonus)
-                    stat_names = set(map(lambda x: Stat(x).name, filters.stats))
-                    stat_sq = (
-                        db_session.query(ModelSetBonus.set_id, ModelSetBonus.stat)
-                        .group_by(ModelSetBonus.set_id, ModelSetBonus.stat)
-                        .filter(
-                            ModelSetBonus.stat.in_(stat_names), ModelSetBonus.value > 0
-                        )
-                    ).subquery()
-                    bonus_sq = (
-                        db_session.query(
-                            ModelSetBonus.set_id,
-                            func.count(distinct(stat_sq.c.stat)).label(
-                                "num_stats_matched"
-                            ),
-                        )
-                        .join(stat_sq, ModelSetBonus.set_id == stat_sq.c.set_id)
-                        .group_by(ModelSetBonus.set_id)
-                        .subquery()
+        level_sq = (
+            db.session.query(ModelItem.set_id, func.max(ModelItem.level).label("level"))
```

Comment:
remind me on the distinction of db.session and session_scope. i see both being used